x-pack/plugins/ml/server/routes/apidoc_scripts/template.md

```diff
@@ -0,0 +1,143 @@
+<a name="top"></a>
+# <%= project.name %> v<%= project.version %>
+
+<%= project.description %>
```

Comment:
I think we could remove that line as well, looks a bit lost IMO (it's the 2nd line `ML Kibana API`):
![image](https://user-images.githubusercontent.com/1945390/81945896-f6055d80-95fe-11ea-81ce-593e5d40133e.png)

Reply:
It uses a projects description, I just put a dummy string "ML Kibana API", could be some valuable info instead 🙂

Reply:
Agreed, some actual description might be nice. Maybe something like
```
This documentation covers the Kibana internal endpoints of the Machine Learning app.
```
@peteharverson any other idea?

Reply:
Or maybe 

```
This documentation provides details of the REST API provided by the Machine Learning (ML) Kibana plugin.
```

Reply:
Or another suggestion, thanks to @szabosteve:

```
This is the documentation of the REST API provided by the Machine Learning Kibana plugin. Each API is experimental and can include breaking changes in any version.
```

Reply:
Updated in https://github.com/elastic/kibana/pull/66567/commits/80d55c45299d337b1ff06d82625ea8a45399b5eb